src/main/java/com/study_buddy/study_buddy/controller/LoginController.java

```diff
@@ -50,24 +61,85 @@ public Map<String, String> register(@RequestBody Register data) {
         String lastName = data.getLastName();
         String hashedPassword = data.getHashedPassword();
         StudyRole studyRole = data.getStudyRole();
-        Map<String, String> returnData = Map.of("firstName", firstName, "lastName", lastName, "email", email,
-                "hashedPassword", hashedPassword.toString(), "studyRole", studyRole.toString());
-
-        // TODO: Save this user to the database
-       /* User user = new User(email, hashedPassword, firstName, lastName, studyRole);
+        System.out.println(hashedPassword.toString());
+        // Example logic for generating a JWT after registering a user
+        JwtService jwtService = new JwtService();
+        String token = jwtService.generateToken(data.getEmail());  // Implement JWT generation
```

Comment:
I would remove these 2 comments as they seem to be stale (JWT generation is implemented and this is not example logic anymore)